docs/examples/notebooks/learn/TestStatistics.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>the phrasing for "bounded/unbounded" is odd to me, given we're bounding in both cases.</p><p> </p><p>What if we described it as a model with POIs bounded from below (<code>&gt; 0</code>) and otherwise allowed to float to negative values?</p><p> </p><p>Additionally, introduce the convention that the test statistics bounded from below will be denoted <code>tilde</code>  and those that are unbounded aren't?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scikit-hep/pyhf/pull/1014/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>